thoth-export-server/src/csv/csv_thoth.rs

```diff
@@ -0,0 +1,498 @@
+use csv::Writer;
+use serde::Serialize;
+use std::io::Write;
+use thoth_api::errors::ThothResult;
+use thoth_client::{
+    SubjectType, Work, WorkContributions, WorkFundings, WorkIssues, WorkLanguages,
+    WorkPublications, WorkPublicationsPrices, WorkSubjects,
+};
+
+use super::{CsvCell, CsvRow, CsvSpecification};
+
+pub(crate) struct CsvThoth;
+
+#[derive(Debug, Serialize)]
+struct CsvThothRow {
```

Comment:
Again, `reference` is missing.

Reply:
Let's leave it like that for the time being